From a948bf4661b0f636cafb05d6b178797758850fa9 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 22 May 2018 19:08:08 +0300
Subject: [PATCH 0635/1051] dts: marvell: Add AP SPI interface on AP810

Add support for AP SPI0 in AP810 DTSI file

Change-Id: I962402302003299c2b573a089f5c41fa3a129e36
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap810.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap810.dtsi b/arch/arm64/boot/dts/marvell/armada-ap810.dtsi
index 9606c3aecefd..5d62ef9e0f8a 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap810.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap810.dtsi
@@ -110,6 +110,16 @@
 				status = "disabled";
 			};
 
+			AP810_LABEL(spi0): spi@510600 {
+				compatible = "marvell,armada-380-spi";
+				reg = <0x510600 0x50>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				interrupts = <GIC_SPI 26 IRQ_TYPE_LEVEL_HIGH>;
+				clocks = <&AP810_LABEL(clk) 6>;
+				status = "disabled";
+			};
+
 			AP810_LABEL(gicp): gicp@3000040 {
 				compatible = "marvell,ap806-gicp";
 				reg = <0x3000040 0x10>;
-- 
2.17.1

